packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -73,25 +74,16 @@ export interface InputPromptProps {
 }
 
 // The input content, input container, and input suggestions list may have different widths
-export const calculatePromptWidths = (terminalWidth: number) => {
-  const widthFraction = 0.9;
+export const calculatePromptWidths = (mainContentWidth: number) => {
```

Comment:
this calculation was overly complex. the input prompt was just supposed to be the main area width.